.orchestra/config/components/python.yml

```diff
@@ -60,6 +60,8 @@ builds:
       - openssl
       - readline
       - sqlite
+      - libb2
+      - xz
```

Comment:
F-, I wonder how is this possible? We should now be using `--sysroot` properly which should lead to failures if a dependency is missing in orchestra.

How did you detected these were missing?
Are you sure these are not optional deps?

This would need further investigation, but not today.

Reply:
> How did you detected these were missing?

I was looking at python's build log and noticing a lot of log spam complaining that the `_blake2` module was not present. It seems that while it's an optional dependency, python will complain about its absence every time the `hashlib` module is imported. After that I've looked at the build log a bit closer and it seems that there were a couple of other optional libraries that probably got linked to the OS in the past.
The two main ones are the ones added to the PR:
* libb2, since [python prefers this implementation](https://github.com/python/cpython/blob/main/Lib/hashlib.py#L73)
* liblzma, needed for the `lzma` module

As said above, both are optional (together with a couple of other ones), but IMHO their contribution justifies the inclusion

Reply:
I'm fine with including them, I'd be sad if, despite the `--sysroot` efforts, we were still pulling libraries from the host.
But according to what you're saying this is not the case, so I'm happy.